rollback_on_fail.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+rollback () {
+    curl -L -o - https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz | tar -xzv --directory ./
+    python jf_agent-stable/jf_agent/main.py
+}
+
+echo "Retrieving time limit value from JF"
+TIME_LIMIT=$(curl -H "Accept: application/json" -H "Content-Type: application/json" -H "Jellyfish-API-Token: $JELLYFISH_API_TOKEN" https://app.jellyfish.co/endpoints/agent/time-limit | jq .time_limit_string)
+TIME_LIMIT="${TIME_LIMIT:-12}"
+echo "checking for time limit override"
+TIME_LIMIT="${OVERRIDE_TIME_LIMIT:-$TIME_LIMIT}"
+echo "running agent with timelimit $TIME_LIMIT"
```

Comment:
If you're setting an override, may as well not hit the API at all -
```suggestion
echo "checking for time limit override"
if [[ -v OVERRIDE_TIME_LIMIT ]]; then
	# May want to do some format validation here, I leave that to the reader
	echo "Time limit override detected, agent will timeout at ${OVERRIDE_TIME_LIMIT}"
	TIME_LIMIT="${OVERRIDE_TIME_LIMIT}"
else
	TIME_LIMIT=$(curl -H "Accept: application/json" -H "Content-Type: application/json" -H "Jellyfish-API-Token: $JELLYFISH_API_TOKEN" https://app.jellyfish.co/endpoints/agent/time-limit | jq .time_limit_string)
	TIME_LIMIT="${TIME_LIMIT:-12}"
	echo "Time limit override not detected, agent has contacted Jellyfish to determine timeout (${TIME_LIMIT})"
fi
echo "running agent with timelimit $TIME_LIMIT"
```